src/node_sqlite.cc

```diff
@@ -116,6 +117,51 @@ using v8::Value;
     }                                                                          \
   } while (0)
 
+class AbortError {
+ public:
+  static MaybeLocal<Object> New(
+      Isolate* isolate,
+      const char* message = "The operation was aborted",
+      Local<Value> cause = Local<Value>()) {
+    Local<String> js_msg;
+    Local<Object> error_obj;
+    Local<Context> context = isolate->GetCurrentContext();
+
+    if (!String::NewFromUtf8(isolate, message).ToLocal(&js_msg) ||
+        !Exception::Error(js_msg)->ToObject(context).ToLocal(&error_obj)) {
+      return MaybeLocal<Object>();
+    }
+
+    if (error_obj
+            ->Set(context,
+                  String::NewFromUtf8(isolate, "name").ToLocalChecked(),
```

Comment:
For fixed constants, you'll probably want to use `FIXED_ONE_BYTE_STRING()`

Reply:
Also, just in general, avoid the use of the `ToLocalChecked()` in these.